### ISSUE TITLE
Fail to install nginx

### ISSUE THREAD

**Issue by mkllnk** [openfoodfoundation/ofn-install#198]
Installing a fresh Ubuntu 16.04 server, I got the following error:
```
        "Setting up nginx-common (1.10.3-0ubuntu0.16.04.2) ...", 
        "Setting up nginx-core (1.10.3-0ubuntu0.16.04.2) ...", 
        "Job for nginx.service failed because the control process exited with error code. See \"systemctl status nginx.service\" and \"journalctl -xe\" for details.", 
        "invoke-rc.d: initscript nginx, action \"start\" failed.", 
        "\u001b[0;1;31m*\u001b[0m nginx.service - A high performance web server and a reverse proxy server", 
        "   Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)", 
        "   Active: \u001b[0;1;31mfailed\u001b[0m (Result: exit-code) since Mon 2018-07-16 09:03:29 UTC; 11ms ago", 
        "  Process: 6715 ExecStart=/usr/sbin/nginx -g daemon on; master_process on; \u001b[0;1;31m(code=exited, status=1/FAILURE)\u001b[0m", 
        "  Process: 6714 ExecStartPre=/usr/sbin/nginx -t -q -g daemon on; master_process on; (code=exited, status=0/SUCCESS)", 
        "", 
        "Jul 16 09:03:27 staging.openfoodnetwork.org.au nginx[6715]: nginx: [emerg] bi...", 
        "Jul 16 09:03:28 staging.openfoodnetwork.org.au nginx[6715]: nginx: [emerg] bi...", 
        "Jul 16 09:03:28 staging.openfoodnetwork.org.au nginx[6715]: nginx: [emerg] bi...", 
        "Jul 16 09:03:28 staging.openfoodnetwork.org.au nginx[6715]: nginx: [emerg] bi...", 
        "Jul 16 09:03:28 staging.openfoodnetwork.org.au nginx[6715]: nginx: [emerg] bi...", 
        "Jul 16 09:03:29 staging.openfoodnetwork.org.au nginx[6715]: nginx: [emerg] st...", 
        "Jul 16 09:03:29 staging.openfoodnetwork.org.au systemd[1]: \u001b[0;1;39mnginx.service: Con...\u001b[0m", 
        "Jul 16 09:03:29 staging.openfoodnetwork.org.au systemd[1]: \u001b[0;1;31mFailed to start A ...\u001b[0m", 
        "Jul 16 09:03:29 staging.openfoodnetwork.org.au systemd[1]: \u001b[0;1;39mnginx.service: Uni...\u001b[0m", 
        "Jul 16 09:03:29 staging.openfoodnetwork.org.au systemd[1]: \u001b[0;1;39mnginx.service: Fai...\u001b[0m", 
        "Hint: Some lines were ellipsized, use -l to show in full.", 
        "dpkg: error processing package nginx-core (--configure):", 
        " subprocess installed post-installation script returned error exit status 1", 
        "dpkg: dependency problems prevent configuration of nginx:", 
        " nginx depends on nginx-core (>= 1.10.3-0ubuntu0.16.04.2) | nginx-full (>= 1.10.3-0ubuntu0.16.04.2) | nginx-light (>= 1.10.3-0ubuntu0.16.04.2) | nginx-extras (>= 1.10.3-0ubuntu0.16.04.2); however:", 
        "  Package nginx-core is not configured yet.", 
        "  Package nginx-full is not installed.", 
        "  Package nginx-light is not installed.", 
        "  Package nginx-extras is not installed.", 
        " nginx depends on nginx-core (<< 1.10.3-0ubuntu0.16.04.2.1~) | nginx-full (<< 1.10.3-0ubuntu0.16.04.2.1~) | nginx-light (<< 1.10.3-0ubuntu0.16.04.2.1~) | nginx-extras (<< 1.10.3-0ubuntu0.16.04.2.1~); however:", 
        "  Package nginx-core is not configured yet.", 
        "  Package nginx-full is not installed.", 
        "  Package nginx-light is not installed.", 
        "  Package nginx-extras is not installed.", 
        "", 
        "dpkg: error processing package nginx (--configure):", 
        " dependency problems - leaving unconfigured", 
        "Setting up python3-pyparsing (2.0.3+dfsg1-1ubuntu0.1) ...", 
        "dpkg: dependency problems prevent configuration of python3-certbot-nginx:", 
        " python3-certbot-nginx depends on nginx; however:", 
        "  Package nginx is not configured yet.", 
        "  Package nginx-core which provides nginx is not configured yet.", 
        "", 
        "dpkg: error processing package python3-certbot-nginx (--configure):", 
        " dependency problems - leaving unconfigured", 
        "dpkg: dependency problems prevent configuration of python-certbot-nginx:", 
        " python-certbot-nginx depends on python3-certbot-nginx; however:", 
        "  Package python3-certbot-nginx is not configured yet.", 
        "", 
        "dpkg: error processing package python-certbot-nginx (--configure):", 
        " dependency problems - leaving unconfigured", 
        "Processing triggers for systemd (229-4ubuntu21.2) ...", 
        "Processing triggers for ureadahead (0.100.0-19) ...", 
        "Processing triggers for ufw (0.35-0ubuntu2) ...", 
        "Errors were encountered while processing:", 
        " nginx-core", 
        " nginx", 
        " python3-certbot-nginx", 
        " python-certbot-nginx"
```

I believe that `python-certbot-nginx` expects nginx to be installed. But nginx is installed in the next step. Just changing the order didn't help immediately as dpkg is in an error state with unmet dependencies now. I will experiment more and come up with a pull request.

Question: Why

**Comment by myriamboure** [comment]
Can this be closed now @mkllnk ?

**Comment by Matt-Yorkley** [comment]
Closed by #205
